test/fuzz/yaml/yaml.go

```diff
@@ -20,10 +20,51 @@ limitations under the License.
 package yaml
 
 import (
+	"bytes"
+
 	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	sigyaml "sigs.k8s.io/yaml"
 )
 
+// FuzzDuration is a fuzz target for unmarshaling Duration defined in "k8s.io/apimachinery/pkg/apis/meta/v1".
+// This target also checks that the unmarshaled result can be marshaled back to the input.
+func FuzzDuration(b []byte) int {
```

Comment:
@mikedanese @BrendanSChang 
if we want to fuzz a lot of these types from core/v1 or meta/v1, does it make sense to start organizing them in separate files?

Reply:
I'll defer to @mikedanese on this one

Reply:
We aren't to the point where this is unmanagable. I hope that we get to that point (by increasing coverage), but I think we can cleanup in a followup.